bundles/org.openhab.binding.etherrain/src/main/java/org/openhab/binding/etherrain/internal/api/EtherRainStatusResponse.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.etherrain.internal.api;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link EtherRainStatusResponse} is a encapsulation of responses from the EtherRain
+ *
+ * @author Joe Inkenbrandt - Initial contribution
+ */
+public class EtherRainStatusResponse {
+    private final Logger logger = LoggerFactory.getLogger(EtherRainStatusResponse.class);
+
+    private String uniqueName;
```

Comment:
I work a lot in Java EE environments professionally, so adhering to the POJO methodology is kinda second nature to me. Since I need to maintain code, I will leave it as is.